plugin.video.ahltv/resources/lib/utils.py

```diff
@@ -0,0 +1,108 @@
+import pytz, time
+import os
+import calendar
+from datetime import date, datetime, timedelta
+from io import StringIO
```

Comment:
You should use `BytesIO` here. `StringIO` is meant for textual data, not binary.

Reply:
This has been updated to import and use BytesIO